### ISSUE TITLE
Feed worldping back to Prometheus so that we can create alert rules

### ISSUE THREAD

**Issue by einari** [dolittle-platform/home#107]
We need to be alerted when something is unreachable from Worldping - without having to have 2 different ways of setting up alerts. 
Prometheus + Alertmanager is our way

**Comment by woksin** [comment]
I don't think this is possible to do. I set up this issue #108 for alerting us on teams when something goes down using worldping and grafana

**Comment by einari** [comment]
I'd like to keep it open in case we can do this later - changing the title to investigate
